src/functionalTest/kotlin/com/cognifide/gradle/aem/instance/LocalInstancePluginTest.kt

```diff
@@ -0,0 +1,146 @@
+                                "tool.search-webconsole-plugin"("com.neva.felix:search-webconsole-plugin:1.2.0")
+                            }
+                        }
+                
+                        instanceProvision {
+                            step("enable-crxde") {
+                                description = "Enables CRX DE"
+                                condition { once() && instance.environment != "prod" }
+                                action {
+                                    sync {
+                                        osgiFramework.configure("org.apache.sling.jcr.davex.impl.servlets.SlingDavExServlet", mapOf(
+                                                "alias" to "/crx/server"
+                                        ))
+                                    }
+                                }
+                            }
+                        }
+                        
+                        register("assertIfCrxDeEnabled") {
+                            doLast {
+                                sync {
+                                    osgiFramework {
+                                        val crxDe = getConfiguration("org.apache.sling.jcr.davex.impl.servlets.SlingDavExServlet")
+                                        if(crxDe.properties["alias"] != "/crx/server") {
+                                            throw Exception("CRX/DE is not enabled on this instance!")
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+                """)
+        }
+
+        runBuild(projectDir, "instanceResolve") {
+            assertTask(":instanceResolve")
+        }
+
+        runBuild(projectDir, "instanceSetup") {
+            assertTask(":instanceSetup")
+            assertFileExists(file(".instance/author"))
+            assertFileExists(file(".instance/publish"))
+        }
+
+        runBuild(projectDir, "instanceDown") {
+            assertTask(":instanceDown")
+        }
+
+        runBuild(projectDir, "instanceBackup") {
+            assertTask(":instanceBackup")
+            val localBackupDir = file("build/instanceBackup/local")
+            assertFileExists(localBackupDir)
+            assert(1 == localBackupDir.walk().filter { it.name.endsWith(".backup.zip") }.count())
+        }
+
+        runBuild(projectDir, "instanceDestroy", "-Pforce") {
+            assertTask(":instanceDestroy")
+            assertFileDoesntExist(file(".instance/author"))
+            assertFileDoesntExist(file(".instance/publish"))
+        }
+
+        runBuild(projectDir, "instanceUp") {
+            assertTask(":instanceCreate")
+            assertTask(":instanceUp")
+            assertFileExists(file(".instance/author"))
+            assertFileExists(file(".instance/publish"))
```

Comment:
there should exist signature accepting path in a same way as other assertion methods, please add it and use it here

Reply:
OK